ledger/participant-state/kvutils/src/test/suite/scala/com/daml/ledger/participant/state/kvutils/ConversionsSpec.scala

```diff
@@ -366,134 +249,6 @@ class ConversionsSpec extends AnyWordSpec with Matchers with OptionValues {
           )
         }
       }
-
-      "produce metadata that can be easily parsed" in {
-        forAll(
-          Table[Rejection, String, String => Any, Any](
-            ("rejection", "metadata key", "metadata parser", "expected parsed metadata"),
-            (
-              Rejection.MissingInputState(partyStateKey("party")),
-              "key",
-              TextFormat.parse(_, classOf[DamlStateKey]),
-              partyStateKey("party"),
-            ),
-            (
-              Rejection.RecordTimeOutOfRange(LfTimestamp.Epoch, LfTimestamp.Epoch),
-              "minimum_record_time",
-              LfTimestamp.assertFromString,
-              LfTimestamp.Epoch,
-            ),
-            (
-              Rejection.RecordTimeOutOfRange(LfTimestamp.Epoch, LfTimestamp.Epoch),
-              "maximum_record_time",
-              LfTimestamp.assertFromString,
-              LfTimestamp.Epoch,
-            ),
-            (
-              Rejection.PartiesNotKnownOnLedger(Iterable(party0, party1)),
-              "parties",
-              jsonString => {
-                val objectMapper = new ObjectMapper
-                objectMapper.readValue(jsonString, classOf[java.util.List[_]]).asScala
-              },
-              mutable.Buffer(party0, party1),
-            ),
-          )
-        ) { (rejection, metadataKey, metadataParser, expectedParsedMetadata) =>
-          val encodedEntry = Conversions
-            .encodeTransactionRejectionEntry(
-              submitterInfo,
-              rejection,
-            )
-            .build()
-          val finalReason = Conversions
-            .decodeTransactionRejectionEntry(encodedEntry, v1ErrorSwitch)
-          finalReason.definiteAnswer shouldBe false
-          val actualDetails = finalReasonDetails(finalReason).toMap
-          metadataParser(actualDetails(metadataKey)) shouldBe expectedParsedMetadata
```

Comment:
@mziolekda The removal was motivated this particular setting - we specifically tested only for v1 error codes.
@pbatko-da @tudor-da Could you correct me or confirm that I can remove this test case?

Reply:
:+1: